### PR TITLE
Task 265174:Changes in UI/Powershell To Show Additional Code Coverage checkbox

### PR DESCRIPTION
**Pull request by AshwiniChalla**
added code coverage enabled option to run distributed tests task.

Testing: uploaded task and Verified UI.

### CODE REVIEW

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 [hunk 1]
```diff
@@ -1,10 +1,11 @@
 param(
     [string]$environment,
-	[string]$dropLocation,
+    [string]$dropLocation,
     [string]$sourcefilters,
     [string]$testFilterCriteria,
     [string]$runSettingsFile,
-	[string]$overrideRunParams
+    [string]$codeCoverageEnabled,
+    [string]$overrideRunParams
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: we need to change the patch version of this task

File: Tasks/RunDistributedTests/task.json [hunk 2]
```diff
@@ -77,6 +77,15 @@
             "groupName": "testSettings"
         },
         {
+            "name": "codeCoverageEnabled",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: does the vstest task have this option in same group.

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 [hunk 3]
```diff
@@ -23,6 +25,6 @@ Write-Verbose "Getting the connection object"
 $connection = Get-VssConnection -TaskContext $distributedTaskContext
 
 Write-Verbose "Calling Invoke-RunDistributedTests"
-Invoke-RunDistributedTests -EnvironmentName $environment -SourceFilter $sourcefilters -TestCaseFilter $testFilterCriteria -RunSettingsPath $runSettingsFile -TestRunParams $overrideRunParams -TestDropLocation $dropLocation -Connection $connection
+Invoke-RunDistributedTests -EnvironmentName $environment -SourceFilter $sourcefilters -TestCaseFilter $testFilterCriteria -RunSettingsPath $runSettingsFile -CodeCoverageEnabled $codeCoverageEnabled -TestRunParams $overrideRunParams -TestDropLocation $dropLocation -Connection $connection
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: what happens when the agent is old and task is new and we try to pass this argument?

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 (hunk 1 above)
Comment (AshwiniChalla) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: agreed

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 (hunk 3 above)
Comment (AshwiniChalla) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: It should not effect as we are modifying run settings in the task and sending it to agent.

**Pull request merged**